source/common/json/config_schemas.cc

```diff
@@ -671,6 +671,7 @@ const std::string Json::Schema::ROUTE_ENTRY_CONFIGURATION_SCHEMA(R"EOF(
       "prefix_rewrite" : {"type" : "string"},
       "host_rewrite" : {"type" : "string"},
       "auto_host_rewrite" : {"type" : "boolean"},
+      "auto_host_rewrite_header" : {"type" : "string"},
```

Comment:
Likewise, this schema is for the v1 api and should be reverted.

Reply:
File reverted.